guardian/tools-telemetry-service
# Initialise CDK in new folder

_co-authored-by:_ @tjsilver

## What does this change?

This PR adds a scaffold for the CDK we'll need to generate a lambda and an API gateway. The end goal is an endpoint which we can post telemetry events to, and a service that pushes those events onto a Kinesis stream.

## How to test

`cd` into the cdk folder and run `npm i; npm run test`. The dependencies should be installed, and the tests should pass.

## How can we measure success?

We've got a working CDK scaffold for a stack.

## Have we considered potential risks?

None to mention.
